test/conformance/runtime/cgroup_test.go

```diff
@@ -31,16 +30,6 @@ import (
 	. "knative.dev/serving/pkg/testing/v1"
 )
 
-const (
-	cpuLimit    = 1     // CPU
-	memoryLimit = 128   // MiB
-	cpuRequest  = 0.125 // CPU
```

Comment:
What happened to these default values?

I'm okay with making this parameterizable, but it seems like the default behavior without setting flags shouldn't change (and then we can discuss whether the defaults are sane separately).

Reply:
The defaults are now set in `e2e_flags`. I can change those to contain what the current default values are or these ones here.

Reply:
I would avoid changing the defaults in this PR -- we can have a separate PR to change the defaults if we think the new defaults make more sense.

Reply:
This allows changing one thing at a time, which means that if we discover some knock-on effect of the different sizing (like it causes nightly tests to fail with out of resources), your flags don't get removed with the rollback.